spec/basket_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'basket'
+
+describe Basket do
+
+  describe '#basket' do
+    it 'begins as an empty array' do
+    expect(subject.basket).to be_empty
+    end 
+  end
+
+  describe 'order_total' do 
+    it 'begins with a value of zero' do
+    expect(subject.order_total).to eq 0
+    end
+  end 
+
+  describe 'add_to_basket(item)' do
```

Comment:
Instead of hardcoding your menu item you could create mock let (:menu) {double :menu} and then expect menu.menu to receive ([{"Double double" => 1}])
(I'll check the syntax when I finish review)